h2o-py/h2o/model/dim_reduction.py

```diff
@@ -94,19 +94,17 @@ def proj_archetypes(self, test_data, reverse_transform=False):
         return h2o.get_frame(j["model_metrics"][0]["predictions"]["frame_id"]["name"])
 
 
-    def screeplot(self, type="barplot", **kwargs):
+    def screeplot(self, type="barplot", server=False):
         """
         Produce the scree plot.
 
         Library ``matplotlib`` is required for this function.
 
         :param str type: either ``"barplot"`` or ``"lines"``.
+        :param bool server: if true set server settings to matplotlib and show the graph
```

Comment:
if server is true, show the graph? are you sure?
```suggestion
        :param bool server: if true set server settings to matplotlib and do not show the graph
```

Reply:
Thanks, I completely missed that. I copied it from varimp_plot so I'll fix that as well.